src/modules/blockchainApplication/connection/components/RequestSummary/index.js

```diff
@@ -20,47 +26,79 @@ const getTitle = (key, t) =>
   Object.values(SIGNING_METHODS).find(item => item.key === key)?.title
     ?? t('Method not found');
 
+const getRequestTransaction = (request) => {
+  const { payload, schema } = request.request.params;
+  return decodeTransaction(Buffer.from(payload, 'hex'), schema);
+};
+
 // eslint-disable-next-line max-statements
 const RequestSummary = ({ nextStep }) => {
   const { t } = useTranslation();
+  const { getAccountByAddress } = useAccounts();
   const { events } = useContext(ConnectionContext);
   const [request, setRequest] = useState(null);
+  const [transaction, setTransaction] = useState(null);
+  const [senderAccount, setSenderAccount] = useState(null);
   const { session } = useSession();
+  useDeprecatedAccount(senderAccount);
+  useSchemas();
 
   const approveHandler = () => {
-    const tx = request.params.request.params.rawTx;
-    const moduleCommandID = joinModuleAndCommand({
-      moduleID: tx.moduleID,
-      commandID: tx.commandID,
+    const moduleCommand = joinModuleAndCommand({
+      moduleID: transaction.module,
+      commandID: transaction.command,
+    });
+    const rawTx = elementTxToDesktopTx(transaction, moduleCommand);
+    nextStep({
+      rawTx: {
+        ...rawTx,
+        sendingChain: { chainID: request.chainId.replace('lisk:', '') },
+        recipientChain: { chainID: request.recipientChainID },
+        // @todo Remove composedFees
+        composedFees: {
+          transaction: `${fromRawLsk(transaction.fee)} LSK`,
+        },
+      },
+      selectedPriority: { title: 'Normal', selectedIndex: 0, value: 0 },
     });
-    const binaryTx = convertTxJSONToBinary(
-      tx, moduleCommandID,
-    );
-    const rawTx = elementTxToDesktopTx(binaryTx, moduleCommandID);
-    nextStep({ rawTx });
   };
   const rejectHandler = () => {
     rejectLiskRequest(request);
   };
 
   useEffect(() => {
     const event = events.find(e => e.name === EVENTS.SESSION_REQUEST);
-    if (event) {
-      setRequest(event.meta);
+    if (event?.meta?.params?.request?.params) {
+      setRequest(event.meta.params);
     }
   }, []);
 
+  useEffect(() => {
+    if (request && !transaction) {
+      const tx = getRequestTransaction(request);
+      setTransaction(tx);
+      const address = extractAddressFromPublicKey(tx.senderPublicKey);
+      const account = getAccountByAddress(address);
+      // @todo if account doesn't exist, we should inform the user that the tx can't be signed
```

Comment:
please create a ticket for it

Reply:
There's already a ticket. 
https://github.com/LiskHQ/lisk-desktop/issues/4519